host_guest/cyclodextrin_derivatives/hosts/README.md

```diff
@@ -1,3 +1,9 @@
 # Hosts
 
-Provided are the hosts for this challenge in PDB and ChemDraw format, including beta cyclodextrin as well as the series of modified cyclodextrins.
+Provided are the hosts for this challenge in PDB and ChemDraw (.cdx) format, including beta cyclodextrin as well as the series of modified cyclodextrins.
+`MGLab8.pdb, MGLab9.pdb, MGLab23.pdb, MGLab24.pdb, MGLab34.pdb, MGLab35.pdb, MGLab36.pdb` : PDB host files provided by Katy Kellett April 26th, 2019.
```

Comment:
In future commits, will look nicer if you put each in single quotes. But this will work for now.